src/TransformerGo.cs

```diff
@@ -146,13 +146,27 @@ private void TransformEnumTypes(CodeModelGo cmg)
             {
                 if (em.Values.Where(v => topLevelNames.Contains(v.Name) || CodeNamerGo.Instance.UserDefinedNames.Contains(v.Name)).Any())
                 {
-                    em.HasUniqueNames = false;
+                    foreach (var v in em.Values)
+                    {
+                        v.Name = em.Name + v.Name;
+                    }
                 }
                 else
                 {
                     topLevelNames.UnionWith(em.Values.Select(ev => ev.Name));
                 }
             }
+
+            var modelList = new List<EnumValue>();

```

Comment:
Seems like modelList just evaporates after the following foreach block, why is it needed?

Reply:
It is needed in the `GetUnique` func

Reply:
Ugh I missed it, makes sense.